### PR TITLE
Improvements to support Debian based servers

### PR DESCRIPTION
**Pull request by erral**
- Explicitely install the `sudo` package
- Use ansible variables to decided if this is a Debian or Ubuntu server
- Document that one needs to install Debian or Ubuntu on the server

### CODE REVIEW

File: project/{{ cookiecutter.__folder_name }}/devops/README.md [hunk 1]
```diff
@@ -51,6 +51,12 @@ STACK_NAME={{ cookiecutter.__devops_stack_name }}
 
 Note: The `.env` file is included in `.gitignore`, ensuring environment-specific configurations aren't pushed to the repository.
 
+
+### Server installation
+
+You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH and install python3 there.
```
Comment (stevepiercy) [inline project/{{ cookiecutter.__folder_name }}/devops/README.md:57]: This sentence still does not make sense, even after I cleaned up a part of it.

```suggestion
You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 there.
```

Comment (stevepiercy) [reply project/{{ cookiecutter.__folder_name }}/devops/README.md:57]: Perhaps "You need to specify Ubuntu or Debian in..."?

And is "there" in the file `.env`?

I see `ansible_lsb.id` below. Does that figure into this?

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: project/{{ cookiecutter.__folder_name }}/devops/README.md (hunk 1 above)
Comment (erral) [reply project/{{ cookiecutter.__folder_name }}/devops/README.md:57]: Yes, the `ansible_lsb.id` figures out if we are using Debian or Ubuntu in the server. No need to set it anywhere.

Comment (stevepiercy) [reply project/{{ cookiecutter.__folder_name }}/devops/README.md]: OK. How do we fix this sentence? It still doesn't make sense.

File: project/{{ cookiecutter.__folder_name }}/devops/README.md [hunk 2]
```diff
@@ -51,6 +51,12 @@ STACK_NAME={{ cookiecutter.__devops_stack_name }}
 
 Note: The `.env` file is included in `.gitignore`, ensuring environment-specific configurations aren't pushed to the repository.
 
+
+### Server installation
+
+You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 there.
```
Comment (stevepiercy) [inline project/{{ cookiecutter.__folder_name }}/devops/README.md:57]: This still doesn't make sense:

"You need to Ubuntu or Debian in {{ cookiecutter.hostname }}"

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I'm not sure how to fix this sentence.

File: project/{{ cookiecutter.__folder_name }}/devops/README.md (hunk 2 above)
Comment (erral) [reply project/{{ cookiecutter.__folder_name }}/devops/README.md:57]: What about, `you need an Ubuntu or Debian based system in {{cookiecutter.hostname}}`

Comment (stevepiercy) [reply project/{{ cookiecutter.__folder_name }}/devops/README.md:57]:
```suggestion
You need either a Ubuntu or Debian based system for {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 on that system.
```

Comment (stevepiercy) [reply project/{{ cookiecutter.__folder_name }}/devops/README.md]: I think this does it. This is good.

**Review by stevepiercy — APPROVED** [review]

**Review by ericof — APPROVED** [review]

**Pull request merged**